src/mps/dmrg.jl

```diff
@@ -278,7 +278,7 @@ function dmrg(PH, psi0::MPS, sweeps::Sweeps; kwargs...)
         end
 
         energy = vals[1]
-        phi::ITensor = vecs[1]
+        phi::ITensor = itensor(storagetype(phi)(convert(typeof(data(phi)),data(vecs[1]))), inds(phi))
```

Comment:
Note sure why this change is needed.

Reply:
Again this is related to the CUDA `UnifiedMemory` issue. The solve above returns `vecs[1]` which is a `CuArray{ElT, N, DeviceBuffer}`.